django/test/runner.py

```diff
@@ -784,8 +784,8 @@ def reorder_suite(suite, classes, reverse=False):
     bins = [OrderedSet() for i in range(class_count + 1)]
```

Comment:
Given the change in the for loop below; `class_count` is only used in one place, so it might also be simpler to get rid of it altogether. This Seems (slightly) more readable to my mind - might be a matter of opinion though 🤔 
```suggestion
    bins = [OrderedSet() for i in range(len(classes) + 1)]
```

Reply:
Yes, I had noticed that. I'm actually planning a PR after PR #14076 is merged, which will cause the need to call `len(classes)` to go away completely. But that depends on #14076. The change here was independent of those other changes, and I didn't want to needlessly change the same line twice.